catalyst_voices/packages/libs/catalyst_cardano_serialization/test/test_utils/selection_utils.dart

```diff
@@ -0,0 +1,403 @@
+import 'dart:math';
+import 'package:catalyst_cardano_serialization/catalyst_cardano_serialization.dart';
+import 'package:convert/convert.dart';
+
+/// A utility class for generating random data and performing selection-related
+/// operations for testing purposes.
+///
+/// This class is used only in testing and resides under the `./test` folder.
+sealed class SelectionUtils {
```

Comment:
Does the class need to be extended? If not consider making it final

```suggestion
final class SelectionUtils {
```